docs/querying-data.rst

```diff
@@ -0,0 +1,157 @@
+Querying data in Kingfisher Process
+====================================
```

Comment:
Can remove "in Kingfisher Process"

Reply:
Done in https://github.com/open-contracting/kingfisher-process/pull/220/commits/f147f690aec9099b7e405943b55a780027dc9ee4